src/nextgenMediaImageResolutions/index.ts

```diff
@@ -13,32 +13,46 @@ import {
 } from '../nextgen/nextgen_generator';
 import { sepolia } from '@wagmi/chains';
 import { loadEnv } from '../secrets';
-import { CloudFrontClient } from '@aws-sdk/client-cloudfront';
 import { sendDiscordUpdate } from '../notifier-discord';
 
 const logger = Logger.get('NEXTGEN_IMAGES_LOOP');
 
 let s3: S3Client;
-let cloudfront: CloudFrontClient;
 
 async function setup() {
   s3 = new S3Client({ region: NEXTGEN_BUCKET_AWS_REGION });
-  cloudfront = new CloudFrontClient({ region: NEXTGEN_BUCKET_AWS_REGION });
+}
+
+enum Resolution {
+  'thumbnail',
+  '0.5k',
+  '1k',
+  '2k',
+  '4k',
+  '8k',
+  '16k'
 }
 
 const START_INDEX = 10000000000;
 const END_INDEX = 10000000999;
 
-const BATCH_SIZE = 30;
+const BATCH_SIZE = 60;
 
 export const handler = async () => {
   const start = Time.now();
   logger.info(`[RUNNING]`);
   await loadEnv([]);
   setup();
-  const resolutions = ['2k', '4k', '8k'];
-  for (let resolution of resolutions) {
-    const isFinished = await findMissingImages(resolution);
+
+  const resolutions = [Resolution['thumbnail'], Resolution['0.5k']];
+
+  for (let resolutionEnum of resolutions) {
+    const resolution = Resolution[resolutionEnum].toString();
```

Comment:
Why convert it to string here? Why not pass it as a "Resolution" type to other functions?

Reply:
Only reason why you need it as string in certain places is to uppercase it for log/discord messages. 
1. I don't see why it has to be uppercased for those. 
2. Strong narrow types are always better, it eliminates the possibility of someone using those functions with unintended values.